pkg/config/config.go

```diff
@@ -134,13 +137,29 @@ func (cfg *Config) SetupLogger() {
 	ctrlrt.SetLogger(zap.New(zap.UseFlagOptions(&zapOptions)))
 }
 
+// PopulateAccountIdIfMissing uses sts GetCallerIdentity API to find
+// AWS AccountId when Config.AccountId is empty
+func (cfg *Config) PopulateAccountIdIfMissing() error {
+	if cfg.AccountID == "" {
+		// use sts to find AWS AccountId
+		session := session.Must(session.NewSession())
+		client := sts.New(session)
+		res, err := client.GetCallerIdentity(&sts.GetCallerIdentityInput{})
+		if err == nil {
+			cfg.AccountID = *res.Account
+		}
+		return err
+	}
+	return nil
+}
```

Comment:
Instead of requiring a change to the code-generator, how about just calling `defaultAWSAccountID` from the `Validate()` method below?